apps/prairielearn/src/tests/editorGetNamesForAdd.test.ts

```diff
@@ -0,0 +1,125 @@
+import { assert } from 'chai';
+import { step } from 'mocha-steps';
+
+import { CourseInstanceAddEditor } from '../lib/editors.js';
+
+const editor = new CourseInstanceAddEditor({
+  locals: {},
+  short_name: '',
+  long_name: '',
+  start_access_date: '',
+  end_access_date: '',
+});
+
+describe('Editor getNamesForAdd Tests', () => {
+  step('No specified short_name and long_name should default them to New_1', () => {
```

Comment:
Our use of `mocha-steps` in the integration tests is only because we use each step to build up state that the following tests rely on. For unit tests like this that don't share state, it's generally better to just use `it(...)`. See here for an example: https://github.com/PrairieLearn/PrairieLearn/blob/master/apps/prairielearn/src/lib/json.test.ts

Reply:
Got it! I made this change in `editors.test.ts` (the new file for these tests)